node/src/node.rs

```diff
@@ -1,8 +1,7 @@
-//! High-level integration of [`P2p`], [`Store`], [`Syncer`].
+//! Node that connects to Celestia's P2P network.
 //!
-//! [`P2p`]: crate::p2p::P2p
-//! [`Store`]: crate::store::Store
-//! [`Syncer`]: crate::syncer::Syncer
+//! Node will connect to Celestia's P2P network then will proceed
```

Comment:
```suggestion
//! Upon creation, `Node` will try to connect to Celestia's P2P network and then proceed
```
slightly more specific about the timeline of events, wdyt?

Reply:
looks good